tests/modules/mcuboot/direct_xip/prj.conf

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_ZTEST=y
+
+CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_BOOT_BUILD_DIRECT_XIP_VARIANT=y
+
+# Increase this and rebuild to verify that the DIRECT_XIP variant is booted.
+CONFIG_MCUBOOT_IMAGE_VERSION="1.1.1+1"
```

Comment:
This has to be tested manually? Or have there been created a JIRA ticket to test for doing a double build etc. in their test plan.

Reply:
So far it has been tested manually. At some point, we should add basic automatic testing of the Direct XIP feature to the bootloader test suite.

The typical test scenario:
1. Build the project and flash it with the default version.
2. Increase the image version and build the project again.
3. Flash the primary/secondary slot with the new signed image.
4. Verify that the slot with the newest version is booted (in this case secondary slot).